relate/settings.py

```diff
@@ -255,7 +261,13 @@
 
 # {{{ celery config
 
-BROKER_URL = 'django://'
+RELATE_SITE_NAME = gettext_noop("RELATE")
+RELATE_CUTOMIZED_SITE_NAME = local_settings.get("RELATE_CUTOMIZED_SITE_NAME")
+if RELATE_CUTOMIZED_SITE_NAME is not None and RELATE_CUTOMIZED_SITE_NAME.strip():
+    RELATE_SITE_NAME = RELATE_CUTOMIZED_SITE_NAME
```

Comment:
Could you move this outside of the "celery config" section?

Reply:
Ops.., my bad. Updated.